### PR TITLE
Fix small bug when deleting alerts

### PR DESCRIPTION
**Pull request by chris-bes**
### Issue:
https://github.com/beyondessential/tupaia-backlog/issues/2838#issuecomment-848457933

### Changes:

- Fixed small bugs when deleting/restoring alerts

### CODE REVIEW

File: packages/psss/src/containers/Modals/DeleteAlertModal.js [hunk 1]
```diff
@@ -31,20 +31,25 @@ export const DeleteAlertModal = ({ isOpen, handleClose, alertId }) => {
...
+  }, [setStatus, handleClose]);
+
   if (status === STATUS.SUCCESS) {
     return (
       <SuccessModal
         isOpen={isOpen}
         title="Delete Alert"
         mainText="Alert successfully deleted"
-        handleClose={handleClose}
+        handleClose={onClose}
```
Comment (kael89) [inline packages/psss/src/containers/Modals/DeleteAlertModal.js:45]: A common convetion in React is to name the prop `on{{Event}}` and the handler `handle{{Event}}`:

```suggestion
        onClose={handleClose}
```

I could have commented this in an earlier PR but I just noticed

**Review by kael89 — APPROVED** [review]
Looks good Chris! I left a comment about a convention that shouldn't be too hard to change, but it's totally optional at this stage

File: packages/psss/src/containers/Modals/DeleteAlertModal.js (hunk 1 above)
Comment (chris-bes) [reply packages/psss/src/containers/Modals/DeleteAlertModal.js]: I can never say no to following convention 😬 . However I noticed there are other places still doing this, but I've just changed what I've touched 👍

Comment (kael89) [reply packages/psss/src/containers/Modals/DeleteAlertModal.js]: Thanks for taking the time to do so! 💯

**Pull request merged**